.github/workflows/agentic-tests-nightly.yml

```
@@ -0,0 +1,360 @@
+
+            Be thorough and demonstrate expertise.
+          allowed_tools: "Read"
+          max_turns: 5
+
+      - name: LLM-as-a-Judge validation
+        if: github.event.inputs.include_judge != 'false'
+        uses: anthropics/claude-code-action@v1
+        id: judge
+        with:
+          anthropic_api_key: ${{ secrets.ANTHROPIC_API_KEY }}
+          prompt: |
```

actionlint (1.7.9)

property "response" is not defined in object type {branch_name: string; execution_file: string; github_token: string; structured_output: string}

(expression)